apisix/plugins/jwt-auth.lua

```diff
@@ -53,7 +53,7 @@ local consumer_schema = {
             enum = {"HS256", "HS512", "RS256"},
             default = "HS256"
         },
-        exp = {type = "integer", minimum = 1},
+        exp = {type = "integer", minimum = 1, default = 86400},
```

Comment:
and we need to add a test case to check the default value, we need to sure it works.

and we need a test case for this case: https://github.com/apache/apisix/issues/2649

we need to sure we fixed this bug.

Reply:
Do you mean that when jwt expires (`exp` = `default` value), need to verify that jwt is invalid?